### DISCUSSION TITLE
Is there a way to init as fullscreen when launching keyboard shortcut?

### DISCUSSION THREAD

**Discussion by rafa-ulloa** [kovidgoyal/kitty#3319] [Q&A] [answered]
i know you can `kitty --start-as fullscreen` but there is no option in the config file to launch kitty as fullscreen by default. i was wondering: i mapped `Window + T` to launch Kitty (I have Pop!_OS). Is there a way to add in the `--start-as fullscreen` option when launching kitty from keyboard shortcut? or any other way to make it work?

**Comment by kovidgoyal** [comment, marked as answer]
Assuming your window manager doesnt accept arguents to commands for launch, create a wrapper script to do it instead.
